{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -337,8 +337,13 @@ def _env_get_required(setting_name):
 #
 # Rollbar logging config
 #
+if IN_PROD:
+    ROLLBAR_ACCESS_TOKEN = _env_get_required("ROLLBAR_ACCESS_TOKEN")
+else:
+    ROLLBAR_ACCESS_TOKEN = os.environ.get("ROLLBAR_ACCESS_TOKEN")
```

Comment:
This works to unblock the initial bootstrap steps.
But I don't know if there are side effects to running Review Apps and Staging that have Rollbar set up with a bad token.